webapp/src/layouts/Dashboard.js

```diff
@@ -62,24 +63,36 @@ const Dashboard = ({ children, width, ual }) => {
 
   useEffect(() => {
     if (routes.some((route) => route.path === location.pathname)) {
+      const managementCardTitle = lacchain.dynamicTitle || ''
+
       setRouteName({
         dynamicTitle:
           location.pathname === '/management'
-            ? lacchain.dynamicTitle
+            ? managementCardTitle
             : t(`${location.pathname}>heading`),
         networkTitle: location.pathname === '/' ? eosConfig.networkLabel : '',
-        pathname: location.pathname
+        pathname: location.pathname,
+        pageTitle:
+          location.pathname === '/'
+            ? eosConfig.networkName
+            : `${eosConfig.networkName}${location.pathname}>title`
```

Comment:
@tetogomez we don't need to add the network name as a prefix because we already able to use custom translations for specific networks please take a look to the above comments